frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "next lint",
     "test:ts": "tsc --noEmit",
     "prepare": "husky install",
-    "api:ts": "openapi-typescript https://dharmamitra.org/api-db/openapi.json -o src/codegen/api/v2.d.ts"
+    "api:ts": "openapi-typescript && eslint ./src/codegen/api/*.ts --fix"
```

Comment:
awesome! But doesn't our existing githook already take care of the linting fixes?

Reply:
ah, I see! I'll try to fix this husky setup soon.

But husky should also run eslint --fix.